File: episode-16-octave/fizzbuzz.m
#!/usr/bin/env octave

vector = 1:100
indexes = 1 + !mod(vector, 3) + 2*!mod(vector, 5)

for i=vector
  o = {i, "Fizz", "Buzz", "FizzBuzz"};
  ai = indexes(i);
  disp(o{ai})
endfor

File: episode-21-clojure/fizzbuzz.clj
#!/usr/bin/env clojure -M

(defn fizzbuzz [n]
  (if (= (mod n 15) 0)
    "FizzBuzz"
    (if (= (mod n 3) 0)
      "Fizz"
      (if (= (mod n 5) 0)
        "Buzz"
        n))))

(doseq [i (range 1 101)]
  (println (fizzbuzz i)))
